ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/Impl/Query.hs

```diff
@@ -92,7 +92,7 @@ getPastLedger ::
 getPastLedger CDB{..} = LgrDB.getPastState cdbLgrDB
 
 getHeaderStateHistory ::
-     IOLike m
+     (LedgerSupportsProtocol blk, IOLike m)
```

Comment:
Figure out why

Reply:
Removed!

Was caused by the unnecessary `Measured` constraint on `FingerTree.fmap'`. We switched to the constraint-less [`unsafeFmap`](https://github.com/input-output-hk/cardano-prelude/commit/906d5f2531cee5d70960413c7795c2cee8566cb7) for `AnchoredSeq.bimapPreservingMeasure`.

Why exactly `LedgerSupportsProtocol`?
```haskell
ledgerDbBimap ::
     (Anchorable (WithOrigin SlotNo) a b, GetTip l)
  => ..

instance IsLedger (LedgerState blk) => GetTip (ExtLedgerState blk) where

instance ( IsLedger (LedgerState  blk)
         , LedgerSupportsProtocol blk
         )
      => IsLedger (ExtLedgerState blk) where
```